### PR TITLE
fix:[#HOTFIX] Collapse all filters by default

### LINKED ISSUE #46: Collapse all filters by default

### PR DESCRIPTION
**Pull request by goreck888**
closes #46

### CODE REVIEW

**Comment by Marcelinna** [comment]
For e2e test you have to click on filter name and expand filer list. 
cy.get(#scientific_domains-filter).click()
image

**Comment by Marcelinna** [comment]
1. Go to https://marketplace-9.docker-fid.grid.cyf-kr.edu.pl/services
2. Expand some filters
image
3. Close Tab
4. Go to https://marketplace-9.docker-fid.grid.cyf-kr.edu.pl/services
5. See the problem - still, the same filter is expanded
image
7. Close browser 
8. Open the browser and go again https://marketplace-9.docker-fid.grid.cyf-kr.edu.pl/services
9. See the problem - still, the same filter is expanded
image
 

@AlicjaSwierad Should the filters be collapsed again after closing a tab or browser?

**Comment by AlicjaSwierad** [comment]
yes, filters should be collapsed again after closing a tab or browser?

**Comment by goreck888** [comment]
Do you want to throw out the feature keeping filter state in cookies? IMHO is a feature, not a bug :)

**Comment by Michal-Kolomanski** [comment]
> @AlicjaSwierad @Marcelinna Do you want to throw out keeping filter state in cookies? IMHO is a feature, not a bug :)

That seems like a business decision that can have more implications than expected. I assume that we would like to keep the state of filters in different scenarios. We need to define the scope really precisely

File: app/views/services/filters/_select.html.haml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 = render "services/filters/wrapper", filter: filter do
+  - Rails.logger.debug "FILTER #{filter.inspect}"
```
Comment (Michal-Kolomanski) [inline app/views/services/filters/_select.html.haml:2]: Seems like a debugging artefact - delete

**Review by Michal-Kolomanski — APPROVED** [review]

**Review by Marcelinna — APPROVED** [review]

**Pull request merged**